opencti-platform/opencti-front/src/private/components/Dashboard.jsx

```diff
@@ -908,25 +905,47 @@ const CustomDashboard = ({ dashboard, timeField }) => {
     </Security>
   );
 };
+
+const changeDashboardPref = graphql`
+  mutation DashboardPrefMutation($input: [EditInput]!) {
+    meEdit(input: $input) {
+      dashboard_id
+    }
+  }
+`;
+
 const Dashboard = () => {
   const classes = useStyles();
-  const [view, saveView] = useViewStorage('view-dashboard');
```

Comment:
Can you delete the declaration from ListParameters if it is no longer used ?

Reply:
useViewStorage is no longer used I think you can remove it completely from the code